src/main/java/flight/reservation/plane/Aircraft.java

```diff
@@ -0,0 +1,9 @@
+package flight.reservation.plane;
+
+public interface Aircraft {
```

Comment:
consider making this an abstract class and implement the interfaces methods here. need to add the member variables for this to work.